<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFormClass</class>
 <widget class="QMainWindow" name="MainFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>База заказов EXPERTprint</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>891</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="createOrderBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editOrderBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteOrderBtn">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="printBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="ordersTableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>36</y>
      <width>881</width>
      <height>291</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollPerPixel</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Заказчик</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Оплата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Менеджер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Дизайнер</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Готовность</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>620</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>335</y>
      <width>431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Позиции заказа</string>
    </property>
    <widget class="QTableWidget" name="positionsTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>192</height>
      </rect>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Позиция</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Количество</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Выдача</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="clientCardGroupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>335</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Карта клиента</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="clientCardGridLayout">
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="clientNameLabel">
        <property name="text">
         <string>Клиент</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="clientPhoneLabel">
        <property name="text">
         <string>Телефон</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="clientMailLabel">
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="clientNameLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="clientPhoneLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="clientMailLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="remarkGroupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>431</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Примечание</string>
    </property>
    <widget class="QTextEdit" name="remarkTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>411</width>
       <height>91</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="searchLabel">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Поиск по клиенту:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="changesOrderGroupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>480</y>
      <width>431</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Карта изменений заказа</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <layout class="QVBoxLayout" name="createLoginVBox">
        <item>
         <widget class="QLabel" name="createLoginPositionLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createLoginNameLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="changeAvailabilityLabel">
        <property name="text">
         <string>Изменение (готово/в работе)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="staffLabel">
        <property name="text">
         <string>Сотрудник</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QVBoxLayout" name="editLoginVBox">
        <item>
         <widget class="QLabel" name="editLoginPositionLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="editLoginNameLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout" name="createDateTimeVBox">
        <item>
         <widget class="QLabel" name="createDateLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="createTimeLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="editOrderLabel">
        <property name="text">
         <string>Редактирование заказа</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QVBoxLayout" name="availDateTimeVBox">
        <item>
         <widget class="QLabel" name="availDateLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="availTimeLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QVBoxLayout" name="editDateTimeVBox">
        <item>
         <widget class="QLabel" name="editDateLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="editTimeLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="createOrderLabel">
        <property name="text">
         <string>Создание заказа</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="dateTimeLabel">
        <property name="text">
         <string>Дата / Время</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <layout class="QVBoxLayout" name="availLoginVBox">
        <item>
         <widget class="QLabel" name="availLoginPositionLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="availLoginNameLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1055</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Клиенты</string>
    </property>
    <addaction name="action_addClient"/>
    <addaction name="action_getClientsList"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сотрудники</string>
    </property>
    <addaction name="action_getStaffList"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_addClient">
   <property name="text">
    <string>Добавить клиента</string>
   </property>
  </action>
  <action name="action_getClientsList">
   <property name="text">
    <string>Список клиентов</string>
   </property>
  </action>
  <action name="action_getStaffList">
   <property name="text">
    <string>Список сотрудников</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainForm.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ordersTableWidget</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>726</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createOrderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>772</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>782</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editOrderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot6()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>774</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>772</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteOrderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot7()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>766</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>768</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot5()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>889</x>
     <y>765</y>
    </hint>
    <hint type="destinationlabel">
     <x>731</x>
     <y>766</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_addClient</sender>
   <signal>triggered()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_getClientsList</sender>
   <signal>triggered()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_getStaffList</sender>
   <signal>triggered()</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ordersTableWidget</sender>
   <signal>itemPressed(QTableWidgetItem*)</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot8()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>548</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>585</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchLineEdit</sender>
   <signal>textEdited(QString)</signal>
   <receiver>MainFormClass</receiver>
   <slot>slot9()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
  <slot>slot4()</slot>
  <slot>slot5()</slot>
  <slot>slot6()</slot>
  <slot>slot7()</slot>
  <slot>slot8()</slot>
  <slot>slot9()</slot>
 </slots>
</ui>
